tools/installers/install_gss.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ $# != 0 ]; then
+    echo "Usage: $0"
+    exit 1;
+fi
+
+# check if cupy is installed, if not install it with conda
+if ! command -v gss &>/dev/null; then
+  conda install -yc conda-forge cupy=10.2
```

Comment:
We can't postulate using conda because conda is just a one of supported systems.

Please use pip. Otherwise, please raise an error if conda command is not found,

Reply:
Thanks ! pip will work but then the user must get the correct version of cupy for their cudatoolkit. 
Conda is convenient because it finds the correct version automatically. 
I will raise an error if conda is not found, after all instructions in README.md in this recipe use conda